MAUI/MauiAppBasic/MSALClient/PCAWrapper.cs

```diff
@@ -0,0 +1,108 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Identity.Client;
+
+namespace MauiAppBasic.MSALClient
+{
+    /// <summary>
+    /// This is a wrapper for PCA. It is singleton and can be utilized by both application and the MAM callback
+    /// </summary>
+    public class PCAWrapper
+    {
+        /// <summary>
+        /// This is the singleton used by consumers
+        /// </summary>
+        public static PCAWrapper Instance { get; private set; } = new PCAWrapper();
+
+        internal IPublicClientApplication PCA { get; }
+
+        internal bool UseEmbedded { get; set; } = false;
+
+        /// <summary>
+        /// The authority for the MSAL PublicClientApplication. Sign in will use this URL.
+        /// </summary>
+        private const string _authority = "https://login.microsoftonline.com/common";

```

Comment:
Can you separate the configuration? At least have the config options all be consistent - all public or private consts etc. 

 See `PublicClientApplicationOptions`for another idea of how to do it.

Reply:
And they need a LOT of comments.

Reply:
Added comments for the options and made them consistent as appropriate.